yarn.lock

```diff
@@ -975,19 +975,11 @@ append-transform@^1.0.0:
   dependencies:
     default-require-extensions "^2.0.0"
 
-aproba@^1.0.3, aproba@^1.1.1:
```

Comment:
Are these intended changes?

Reply:
this shouldn’t be necessary, please merge from develop.

Reply:
@marianazangrossi, did you merge from develop yet?

Reply:
Did you push?

Reply:
Yes!

Reply:
Hmm this change shouldn’t be here then.

Reply:
It was replaced by `aproba@^1.1.1:` after the merge. Is it correct?

Reply:
It shouldn't be. I just ran yarn install locally. I'd recommend copy and pasting it directly from the Oppia repo.

Reply:
@marianazangrossi this is still not resolved, ptal!

Reply:
Sorry for the delay @kevinlee12, I've fixed it PTAL